svg/elements/image.json

```diff
@@ -25,7 +25,7 @@
               "version_added": "8"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "10.1"
```

Comment:
I've approved a bunch of these because 10.1 seems to be the right one, but this doesn't quite match your description:

>This PR replaces true/null values with exact version numbers (or false) for Opera and Opera Android for the image SVG element. This sets Opera Android to mirror from upstream.

How come you didn't must do mirror?

Reply:
Setting "mirror" would mirror the data from Chrome Android instead of Opera, so this sort of mirroring must be done manually!